leelastar/network/layers.py

```diff
@@ -48,20 +55,10 @@ def __init__(self, channels, kernel_size, se_ratio):
         super().__init__(
             OrderedDict(
                 [
-                    (
-                        "conv_layer_1",
-                        nn.Conv2d(
-                            channels, channels, kernel_size, padding=1, bias=False
-                        ),
-                    ),
+                    ("conv_layer_1", nn.Conv2d(channels, channels, kernel_size, padding=1, bias=False)),
```

Comment:
My black breaks this line (default configuration), do you use a longer line width?

Reply:
the line is on 120 length

Reply:
-l 120

Reply:
Ok, will make the change in my configuration